scss/os/_alertpopup.scss

```diff
@@ -1,4 +1,5 @@
 // We want alerts to popup above windows and modals
 .c-alert-popup {
+  user-select: text;
```

Comment:
This prompted me to ask @gregroyal what happened to the `.selectable` and `.unselectable` classes in the BS4 update. Some of our views are definitely referencing `.unselectable` even though it does not appear to exist in the compiled CSS.

Rather than just tacking it on here, it would make sense to have some very generic `.selectable` and `.unselectable` utility classes in `scss/os/_utils.scss`, and then use the `selectable` class in the alert popup view template.